src/Gallery/index.tsx

```diff
@@ -18,38 +19,56 @@ const tabs: Array<WhpptTab> = [
 
 export const Gallery: FC<{ onUse?: (image: GalleryItem) => void }> = ({ onUse }) => {
   const { settingsPanel, changeSettingsPanelActiveTab, api, hideSettingsPanel, domain } = useWhppt();
+
   const [items, setItems] = useState<GalleryItem[]>([]);
   const [selected, setSelected] = useState<GalleryItem>(null);
   const [searchQueryTags, setSearchQueryTags] = useState('');
   const [filter, setFilter] = useState('');
-
-  const [loading, setLoading] = useState<'loading' | 'loaded'>('loading');
   const [error, setError] = useState('');
+  const [loading, setLoading] = useState<'loading' | 'loaded'>('loading');
 
   const search = useCallback(() => {
+    setError('');
+
     const tags = splitKeywords(searchQueryTags) || [];
     const type = settingsPanel.activeTab as GalleryFileType;
 
-    return api.gallery
+    const search = api.gallery
       .search({ domainId: domain._id, page: 1, size: 10, type, tags, filter })
       .then(({ items }: { items: GalleryItem[] }) => setItems(items))
       .catch(error => setError(error.message || error));
+
+    return toast.promise(search, {
+      error: 'Image upload failed 🤯',
```

Comment:
Just added the toast notification on error case, so it doesn't pop up on searching and success